rust/gui-client/src-tauri/firezone-client-ipc.service

```diff
@@ -38,6 +38,7 @@ UMask=077
 Environment="FIREZONE_DNS_CONTROL=systemd-resolved"
 Environment="LOG_DIR=/var/log/dev.firezone.client"
 Environment="RUST_LOG=info"
+EnvironmentFile="/etc/default/firezone-client-ipc"
```

Comment:
The ID and token could actually go in this type of file for the headless Clients, if we end up setting them up as deb packages and put the systemd service file in `/usr/lib`